pitch.md

```diff
@@ -1,60 +1,112 @@
+emulators and Twitch's API. A DSL could help people who would have previously
+been excluded from making a "Twitch Plays".
 
 ## Why a language?
 _Why is a **DSL** appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate for multiple reasons. We are focusing on a specific domain,
+listening and executing Twitch chat commands. In addition, we are attempting to
+make that one domain extremely easy while allowing for everything else to be
+hard or impossible. We also want to allow the user to express potentially very
+complex compositions of commands, so the richness of a language is a big boost.
 
 ## Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I was a big fan of the original Twitch Plays Pokemon and I was excited to see
+other people continue the tradition, but no one did. I attribute this significantly
+to the difficulty in creating such a project.
 
 ## Interface (syntax)
 _What kinds of things can the user say in your language. How, generally, do they
 say it? **Be as specific as possible.** You don't need to design syntax; but you
 should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
-important to users and what kinds of values those properties might have._ 
+important to users and what kinds of values those properties might have._
 
+The user would need to specify a couple of things. They would first need to specify
+their channel as the program needs to know which chat to scrape. They should also
+specify how often the game should execute its commands, e.g. every 10 votes or every
+5 seconds. The user might also want to specify how votes are counted, e.g. simple
+plurality or drawn from a hat. Finally, the user would need to specify what do when
+a command is chosen, e.g. when we choose "!jump", the spacebar should be pressed. I
+have ideas for other specifications, but these should be good for a minimum and I
+can extend them later.
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+I'm still not sure how my program should be evaluated. Right now, I'm leaning
+towards a kind of meta programming, where the program is parsed, then used to build
+a valid Python program. This way I can rely on Python's interpreter. This language
+seems like it will use the "Production Rule" model heavily as it involves a lot of
+specifying what a command is and what it should do. I'm sure I'll realize more as
+the project goes on, but right now the only semantic error I can foresee is if the
+user specified the same command multiple times. This can be reported to the user
+on first parse, giving the user the chance to rename the command.
```

Comment:
I'm still slightly confused about what happens when your program runs.  If I'm understanding correctly, your DSL a way of creating a game that additionally defines rules for how to act collaboratively.  Can you give examples of what keywords or syntax would be in the language? What features would your language _not_ include?